kolibri/plugins/learn/assets/src/views/ContentCard/index.vue

```diff
@@ -27,6 +27,7 @@
       </p>
       <div class="footer">
         <CoachContentLabel
+          v-if="hasUserPermissions && !isLearner"
```

Comment:
I can't find this getter in the code. I see a getter in `modules/session.js` called `getUserPermissions` is that what you meant? I don't think that `getUserPermissions` would be useful for this application, however.

Reply:
@nucleogenesis Could you confirm that this is the right getter? I can't find it when searching the code.

Reply:
@jonboiser Great catch - for some reason using a non-existent getter still achieved the correct effect.

In any case - I just pushed it up with the proper getter.

Reply:
Yeah, that looks like a valid reference now, but I'm still not sure if that's the most straightforward way to check for an unauthenticated user... authenticated learners also have `userHasPermissions = false`.

The permissions object is kind of an indirect way to check if a person is authenticated. Is there something more direct we can check inside `core.state.session`?

Reply:
Maybe we can implement a new straightforward getter `isGuest` and check `state.core.session.kind === 'anonymous'`

Reply:
Maybe the simplest way would be to just check that the user is logged in: https://github.com/learningequality/kolibri/blob/release-v0.12.y/kolibri/core/assets/src/state/modules/session.js#L34 and that they are not a learner?

Reply:
Even better!

Reply:
@nucleogenesis Like @rtibbles suggested, use the `isUserLoggedIn` getter instead of `userHasPermissions`